k4MarlinWrapper/k4MarlinWrapper/LcioEventAlgo.h

```diff
@@ -26,19 +26,17 @@
  */
 
 #include <Gaudi/Algorithm.h>
-#include <GaudiKernel/IEventProcessor.h>
+#include <Gaudi/Property.h>
 
-#include <EVENT/LCIO.h>
-#include <MT/LCReader.h>
-
-#include "k4MarlinWrapper/LCEventWrapper.h"
+namespace MT {
+  class LCReader;
+}
 
 class LcioEvent : public Gaudi::Algorithm {
 public:
   explicit LcioEvent(const std::string& name, ISvcLocator* pSL);
-  virtual ~LcioEvent() = default;
-  virtual StatusCode initialize() override final;
-  virtual StatusCode execute(const EventContext&) const override;
+  StatusCode initialize() override final;
+  StatusCode execute(const EventContext&) const override;
```

Comment:
```suggestion
  StatusCode execute(const EventContext&) const override final;
```
This seems to be what we go for in all other places, even though `final` technically already implies `override` so that that is superfluous.